src/Telomare/Parser.hs

```diff
@@ -278,7 +278,7 @@ parseCase = do
   reserved "case" <* scn
   iexpr <- parseLongExpr <* scn
   reserved "of" <* scn
-  lpc <- many $ parseSingleCase <* scn
+  lpc <- some $ try parseSingleCase <* scn
```

Comment:
does this mean an empty case statement is valid?

Reply:
many would allow an empty case statment, some is one or more: https://hackage.haskell.org/package/parser-combinators-1.3.0/docs/Control-Monad-Combinators.html#v:some

Reply:
oh, ok! I always get those two confused